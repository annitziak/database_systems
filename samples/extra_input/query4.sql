SELECT SUM(Student.B) FROM Student GROUP BY Student.B;
