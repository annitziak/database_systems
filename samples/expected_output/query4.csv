1, 200, 50, 33
2, 200, 200, 44
